Magehawks/laravel-nginx-mysql-docker
# Add feature to recieve rss feed informations and send it to opdaen ai…

Step-by-step feature to choose an RSS feed, select an article, and let OpenAI reinterpret it for you.
How To Test
- Check out the repo.
- Create a .env file from the .env.example and add your `OPENAI_API_KEY` there.
- Run `docker-compose up.`
- Go to` http://localhost:8000/feed-form`.
- Add an RSS feed URL (e.g.,` https://www.heise.de/rss/heise-Rubrik-IT.rdf`).
- Choose an article and an interpretation.
- See what OpenAI created for you.
